services/tests/getPceEducationTopics-spec.js

```
@@ -0,0 +1,50 @@
+'use strict';
+
+const _ = require('lodash');
+const pceMetaService = require('../pceMetaService');
+const serviceTestUtils = require('./serviceTestUtils');
+
+describe('lookupEducationTopics asynchronous specs', () => {
+    let context;
+    beforeAll(() => {
+         const res = serviceTestUtils.beforeAllScript();
+
+    });
+
+
+
+    const testReturnsTopicsArray = (res) => {
+        expect(res).toBeDefined();
+        expect(res).toEqual(jasmine.any(Array));
+
+        if (res[0]) {
+            let startsWithA = false;
+            if (res[0].name.toUpperCase().startsWith('A')) {
+                startsWithA = true;
+                expect(startsWithA).toBeTruthy();
+            }
+            else
+                expect(startsWithA).toBeFalsy();
+
+        }
+
+        //console.log(res[0]);
+        if (res.length > 0) {
+            expect(res[0].id).toBeDefined();
+            expect(res[0].name).toBeDefined();
+        }
+
+
+    };
+
+    it('Empty arg should return array of Topics objects', (done) => {
+        pceMetaService.getEducationTopics((err, res) => {
+            if (err) console.log(err);
```

We should probably call `fail()` here instead of just `console.log()`
This and all other unexpected error handler cases in our specs should manually fail out of the test case instead of just printing to console.
Some of our test specs need to be revised to reflect these changes too since we only recently started using `fail()`

---

good point. I copied it from the other spec in the pceMetaService. In fact, all other specs in this folder use console.log. I will modified them as well.